diffkemp/llvm_ir/kernel_llvm_source_builder.py

```diff
@@ -162,6 +164,37 @@ def _enable_asm_features(self):
             except CalledProcessError:
                 pass
 
+    def is_configured(self):
+        """
+        Check if the kernel is properly configured (all necessary options are
+        set).
+        """
+        config_file = os.path.join(self.source_dir, ".config")
+        config_backup_file = os.path.join(self.source_dir,
+                                          ".config-diffkemp-bkup")
```

Comment:
Let's name it `.config.bak`, that's sort of standard.

Reply:
It is renamed, thanks.